test/PackageVersionsTest/VersionsTest.php

```diff
@@ -1,14 +1,21 @@
 <?php
 
+declare(strict_types=1);
+
 namespace PackageVersionsTest;
 
+use OutOfBoundsException;
 use PackageVersions\Versions;
 use PHPUnit\Framework\TestCase;
+use function array_merge;
+use function file_get_contents;
+use function json_decode;
+use function uniqid;
 
 /**
- * @covers \PackageVersions\Versions
- *
  * @uses \PackageVersions\FallbackVersions
+ *
+ * @covers \PackageVersions\Versions
```

Comment:
Some FCQNs were not imported, this looks like Slevomat CS bug.

Reply:
It's actually by design, only `'@var', '@param', '@return', '@throws', '@property', '@property-read', '@property-write'` are analysed.